Source/WarpX.H

```diff
@@ -83,15 +83,11 @@ public:
 
     static void GotoNextLine (std::istream& is);
 
-    // External fields added to particle fields.
-    static amrex::Vector<amrex::Real> B_external_particle;
-    static amrex::Vector<amrex::Real> E_external_particle;
-
     // Initial field on the grid.
     static amrex::Vector<amrex::Real> E_external_grid;
     static amrex::Vector<amrex::Real> B_external_grid;
 
-    // Initialization Type for External E and B
+    // Initialization Type for External E and B on grid and particles
```

Comment:
Is this really for fields and particles? I am confused.

Reply:
only for grid. dint change the comment after changing the code. Thank you for the correction.